charts/artifact-caching-proxy/templates/statefulset.yaml

```diff
@@ -37,7 +37,7 @@ spec:
           runAsUser: 0
           runAsGroup: 0
         command: ['sh', '-c']
-        args: ["chown -Rv {{ .Values.initContainer.chownArgs }}  {{ .Values.paths.cachePath }} && ls -l {{ .Values.paths.cachePath }}"]
```

Comment:
Not sure what does it change?

Previous version uses the `-R` flag (e.g. "recursive"). So *when* the command was triggered, if `lost+found` was present, then it would have been treated.

Reply:
Yup, you confirmed my suspicions, closing this PR.